servers/zms/src/main/java/com/yahoo/athenz/zms/ZMSImpl.java

```diff
@@ -1966,6 +1966,13 @@ BitSet validateDomainSystemMetaStoreValues(Domain domain, DomainMeta meta, final
                     changedAttrs.set(DomainMetaStore.META_ATTR_PRODUCT_ID);
                 }
                 break;
+            case ZMSConsts.SYSTEM_META_BUSINESS_SERVICE:
+                if (ZMSUtils.metaValueChanged(domain.getBusinessService(), meta.getBusinessService())) {
+                    if (!domainMetaStore.isValidBusinessService(domain.getName(), meta.getBusinessService())) {
+                        throw ZMSUtils.requestError("invalid business service for domain", caller);
+                    }
+                    changedAttrs.set(DomainMetaStore.META_ATTR_BUSINESS_SERVICE);
+                }
```

Comment:
let's add a break statement here otherwise we might miss in the future when we add additional case lines.

Reply:
Fixed